### DISCUSSION TITLE
How to get rid of math shortcode tags

### DISCUSSION THREAD

**Discussion by toastedqu** [thuliteio/doks#1237] [Q&A]
I frequently write equations in my blog and the "{{< math >}}" shortcode is quite annoying. We didn't have to use this in older versions of Doks. Is it possible to get rid of this tag and directly use KaTex grammar by changing things in the setting?

**Comment by h-enk** [comment]
You could use a [fenced code block](https://getdoks.org/docs/built-ins/math/#fenced-code-block)

**Reply by toastedqu** [reply]
Thank you for your reply. I did see this on the tutorial but I guess this doesn't solve the problem. What was the motivation for changing the original math equation rendering (without any additional syntax) to the current version of using either a code block or a short code?
